### PR TITLE
Fix pagination after selecting/unselecting budget investment

### PR DESCRIPTION
**Pull request by behraaang**
References
===================
Issue https://github.com/consul/consul/issues/3013

Objectives
===================
Fix pagination link by rendering the specific selected investment on the list

Visual Changes
===================
There won't be any reload to the page when selecting/unselecting the investment 

Notes
===================
There are 3 fail tests on the test suite which are not related to the fix and have been on the fork when I got it

### CODE REVIEW

**Comment by decabeza** [comment]
Hi @behraaang, thanks for the PR! 😌 

There is a failing spec `features/admin/budget_investments_spec.rb:1069` related with this changes. Now the page doesn't refresh and the spec fails, but you can add another `click_button('Filter')` on **L1082** to fix it. 😉 

Also, could you create a new spec to test pagination not fails anymore. 🙏

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,4 @@
 public/sitemap.xml
 public/system/
 /public/ckeditor_assets/
+.idea/
```
Comment (decabeza) [inline .gitignore]: I think this no necessary add this change on the PR 😉

Comment (behraaang) [reply .gitignore]: Hi @decabeza 
I have done the things you have asked, I hope next fix would be with no problem ;).
Thank you

**Comment by voodoorai2000** [comment]
This is great, thank you @behraaang! 🎉

**Pull request merged**